handler.go

```diff
@@ -467,17 +468,24 @@ func populateTmpDir(r *http.Request, rmDir bool, dirpath string) (string, *gz.Er
 
 	// Process files
 	for _, fh := range files {
+		_, params, err := mime.ParseMediaType(fh.Header["Content-Disposition"][0])
```

Comment:
Will all files being processed always have the `Content-Disposition` header? I ask because it might break things if it isn't there every time. I also understand that while the header is technically optional, in practice it might always be there.

Reply:
`Content-Disposition` contains the raw of `fh.Filename`, so I'd expect it's always there.

Reply:
The filename is also optional. I might be making a fuss for no reason but I do want to ensure that we're not breaking things.

Reply:
Do you have any suggestions? Avoid parsing the file if the param is not there? I took this solution from here: https://github.com/teleclimber/Dropserver/commit/a85134d3a48a3ea78b2c3645ce58b52b119aaa0d